### PR TITLE
Make stethoscopes wearable.

### LINKED ISSUE #18140: Stethoscopes should be wearable
Around the player's neck, for example.

### PR DESCRIPTION
**Pull request by Noctifer-de-Mortem**
Fixes #18140

Worn by doctors everyday.

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
>P.S: my game branch is a bit behind CleverRaven's master so I had to manually removed a couple of knifes 0 to hit bonus to prevent crashes.
 
@BevapDin 

I thought GitHub pulled the updated files only?

**Comment by Noctifer-de-Mortem** [comment]
I did it only as a precaution when I noticed some items in tools.json had to hit bonus of 0. After checking around it seems this was redundant but does no harm.

File: data/json/items/tool_armor.json [hunk 1]
```diff
@@ -2129,5 +2129,21 @@
         "encumbrance" : 5,
         "coverage" : 10,
         "material_thickness" : 2
-    }
+    },
+  {
```
Comment (mugling) [inline data/json/items/tool_armor.json]: JSON style, aim for consistency with either existing file or new linter format. The below is neither and also inconsistent with itself.

File: data/json/items/tool_armor.json [hunk 2]
```diff
@@ -2129,5 +2129,21 @@
...
+    "name": "stethoscope",
+    "description": "This is a medical listening tool.  Use it to listen to things.  Closely.",
+    "price": 4000,
+    "material": ["plastic", "aluminum"],
+    "weight": 150,
+    "volume": 1,
+        "covers" : ["TORSO"],
+        "encumbrance" : 1,
+        "coverage" : 5,
+        "material_thickness" : 1
```
Comment (mugling) [inline data/json/items/tool_armor.json]: layer `BELTED`

File: data/json/items/tools.json [hunk 3]
```diff
@@ -6630,7 +6618,6 @@
     "volume": 1,
     "cutting": 16,
     "flags" : ["SHEATH_KNIFE","CHOP"],
-    "to_hit": 0,
```
Comment (mugling) [inline data/json/items/tools.json]: avoid unrelated changes

File: data/json/items/tool_armor.json [hunk 4]
```diff
@@ -2129,5 +2129,22 @@
...
+    "description": "This is a medical listening tool.  Use it to listen to things.  Closely.",
+    "price": 4000,
+    "material": ["plastic", "aluminum"],
+    "weight": 150,
+    "volume": 1,
+        "covers" : ["TORSO"],
+        "encumbrance" : 1,
+        "flags" : ["BELTED"],
+        "coverage" : 5,
+        "material_thickness" : 1
```
Comment (mugling) [inline data/json/items/tool_armor.json]: Don't specify `material_thickness` otherwise it will function as armor
